web/src/pages/admin/fed-admin/DelegateStateAdminForm.test.js

```diff
@@ -10,7 +10,7 @@ import MockAdapter from 'axios-mock-adapter';
 import { plain as DelegateStateAdminForm } from './DelegateStateAdminForm';
 
 const defaultProps = {
-  ffy: '2022',
+  ffy: '2023',
```

Comment:
I think this is fine but we could also use the `thisFFY` util in `/web/src/util/index` to have this update automatically.